ngraph/frontend/frontend_manager/src/frontend_manager.cpp

```diff
@@ -422,3 +429,38 @@ Place::Ptr Place::get_source_tensor(int inputPortIndex) const
 {
     FRONT_END_NOT_IMPLEMENTED(get_source_tensor);
 }
+
+Place::Ptr Place::get_source_tensor(const std::string& inputName, int inputPortIndex) const
+{
+    FRONT_END_NOT_IMPLEMENTED(get_source_tensor);
```

Comment:
should we raise exception or just return nullptr? In my understanding, recent trend is to return nullptr by default to simplify client's code (which should be generic and adding try/catch everywhere in client's code is not convinient).

Reply:
done